docs/api/remote-procedure-calls-evo.md

```diff
@@ -1534,9 +1534,9 @@ Result:
 }
 ```
 
-### ProTx Update Service HPMN
+### ProTx Update Service EvoNode
```

Comment:
The heading here is basically the RPC name with no underscores so let's leave them HPMN
```suggestion
### ProTx Update Service HPMN
```